tezster.js

```diff
@@ -84,8 +84,13 @@ program
     const tezsterManager = require('./tezster-manager');    
     await tezsterManager.loadTezsterConfig();
     const config = tezsterManager.config;
-    for(var i in config.accounts){
-        console.log(config.accounts[i].label + " - " + config.accounts[i].pkh + " (" + config.accounts[i].identity + ")");
+    if(Object.keys(config.accounts).length > 0){
+        for(var i in config.accounts){
+            console.log(tezsterManager.output(config.accounts[i].label + " - " + config.accounts[i].pkh + " (" + config.accounts[i].identity + ")"));
+        }
+    }
+    else{    
+        console.log(tezsterManager.outputError("Accounts are not Available !!"));        
```

Comment:
change message to 'No Account is available !!'

Reply:
done the same.